aristanetworks/ansible-avd
# Refactor(eos_designs)!: Combine the VLAN trunk groups used for MLAG

## Change Summary


Combine the VLAN trunk groups used for MLAG

## Component(s) name

`arista.avd.eos_designs`

## Proposed changes



### Combine the VLAN trunk groups used for MLAG

With AVD version 5.0.0 a single `MLAG` trunk group is used for MLAG and MLAG L3 peerings, including the peerings in VRFs.
Previously MLAG L3 peerings used a separate `LEAF_PEER_L3` trunk group.

The trunk group for MLAG L3 peerings can be reverted if needed:

```diff
+trunk_groups:
+  mlag_l3:
+    name: LEAF_PEER_L3
```

## How to test



- Rerun molecule scenarios to see new default take effect
- Roll back change according to porting-guide suggestion in AVD_LAB for eos_designs_unit_tests and ensure no changes there.

## Checklist

### User Checklist


- N/A

### Repository Checklist
